spec/plugins/data_table/server_side.js

```diff
@@ -35,12 +35,13 @@ describe("DataTable Plugin", function() {
 
   MockResponse.prototype.getWithTotals = function() {
     return {
-      status : 200,
-      responseText : JSON.stringify({
+      status: 200,
+      responseText: JSON.stringify({
         sEcho: this.echo++,
         iTotalRecords: 100,
         iTotalDisplayRecords: 100,
         aaData: [
+          /* eslint-disable camelcase */
```

Comment:
These are going to be super annoying at some point. I wonder if there's a way we can disable this rule for property names on objects.

Reply:
There is a way to disable it for properties: https://eslint.org/docs/rules/camelcase

I'm thinking we should leave it on for properties for now and see how it goes. I had to use a comment in some cases here because those properties mapped to url parameters...

Reply:
We use underscores in property names for model data in Backdraft/Backbone apps. We will get a Hound bark every time that code is updated.

The [`{ "properties": "never" }`](https://eslint.org/docs/rules/camelcase#never) rule disables this warning for properties of objects but keeps the warnings for all other uses of underscores-in-names.

I think that's exactly what we want.

Using `{ "properties": "never" }` you wouldn't have needed to add `eslint-disable camelcase` comments to most (all?) of the places that this comment was added in this PR.

Reply:
alright sounds good, we can use { "properties": "never" }